using-valtimo/forms/creating-forms-in-valtimo.md

```diff
@@ -88,3 +88,81 @@ This behaviour can be changed by setting a toggle in application.yml:
   ```
 
 When set to true, disabled fields will no longer be prefilled or saved. The default value is false.
+
+## Translations
+To display forms in different languages, we can utilize translations. This involves translating all the labels based on their label property, and using a function `instance.t('translationKey')` to display content from the translation file inside the htmlContent.
```

Comment:
`instance.t(...)` feels a bit strange. Can we change that to `instance.translate(...)`?

Reply:
The instance.t() is a form.io function. I'm not sure why they called it .t() instead of translate(). But we can not change that unless it is being changed in form.io library itself. I could check really quick if they maybe also have translate() function, but I got the .t() from the docs so I don't think so.